### ISSUE TITLE
OpenBLAS Warning for FindNumFactors

### ISSUE THREAD

**Issue by MediciPrime** [yanwu2014/swne#16]
Thank you for creating this great visualization tool!  I am currently running through the tutorial and following warning prints out endlessly when I try to run the FindNumFactors function.  

`OpenBLAS Warning : Detect OpenMP Loop and this application may hang. Please rebuild the library with USE_OPENMP=1 option.`

Would you happen to know what this means?

**Comment by MediciPrime** [comment]
I am closing this issue because I have figured out that it have something to do with conda.  I am currently using commit "da1b1ad" to successfully run SWNE, mainly because I am now paranoid and want to use a version that I know has worked for me in the past.  Thank you!  

`devtools::install_github("yanwu2014/swne", ref="da1b1ad")`

**Comment by yanwu2014** [comment]
Hey so I did some digging and it looks like this might have something do with how R was installed and/or the BLAS library it's using. It's a bit of a pain but reinstalling R itself might fix this. This thread has some tips on how to switch the BLAS version R is using as well: https://www.r-bloggers.com/for-faster-r-use-openblas-instead-better-than-atlas-trivial-to-switch-to-on-ubuntu/

**Comment by MediciPrime** [comment]
Totally, I narrowed down the problem to the version of R I had installed on my university's HPC server using Conda.  When I tried installing SWNE on my local computer or by loading the local R module on the HPC server, everything worked fine.  No idea what is wrong with R 3.5.1 from Anaconda's R channel but I will check out your link and see if I can modify the BLAS being used by R to fix the problem on the Conda install.  Thank you for replying even after I closed the issue!
